api/PushSubscription.json

```diff
@@ -321,7 +321,7 @@
               "version_added": "50"
             },
             "edge": {
-              "version_added": null
+              "version_added": "17"
```

Comment:
This is correct and all our Push data should say "17" and not "16" or "16 behind a flag". Nice finding!